backend/app/database/alembic/versions/2025_05_21_1055-6441623a586b_grant_api_user_admin.py

```diff
@@ -35,9 +35,9 @@
 
 def upgrade() -> None:
     session = Session(bind=op.get_bind())
-    api_bot = session.scalar(
-        sa.select(User).filter_by(email="systemaccount@noreply.com")
-    )
+    api_bot = session.execute(
```

Comment:
this statement crashed because at this point of the db migration process the "has_seen_tour" column does not exist yet